shell/server/pre-meteor.js

```diff
@@ -90,7 +96,15 @@ Meteor.startup(function () {
       publicIdPromise = Promise.resolve(id);
     } else {
       // Not a wildcard host. Perhaps it is a custom host.
-      publicIdPromise = lookupPublicIdFromDns(hostname);
+
+      // Skip DNS TXT record resolutions for the major pseudo-TLDs, and avoid
+      // returning any result which might indicate that there's actually a
+      // server listening here.
+      if (hasPseudoTLD(hostname)) {
+        return;
```

Comment:
Unfortunately, returning here will cause no response to ever be sent for this HTTP request, so the user will be left hanging.

You probably want to do something like:

    publicIdPromise = Promise.reject(new Error("Pseudo-TLDs have no DNS records."))

Here's an article about ES6 promises, if you want to learn more about them:

http://www.html5rocks.com/en/tutorials/es6/promises/